.github/workflows/build_conda_pkg.yml

```diff
@@ -0,0 +1,63 @@
+name: Build Conda Package
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
@ParthivNaresh could you please add
```
  pull_request:
    types: [opened, synchronize]
```
and show that this action runs green on this PR before merge?

We've used that syntax in other actions, and I think it means the job will run in this PR but not in other PRs against `main`.

Reply:
Hey, my bad, I clearly do not know enough of the GH actions syntax yet 😆  I was hoping there was a way to run the action in this PR just to prove that it works before merging. If this is too difficult to do (which I think is the case after 5min of googling), no worries, let's merge it and revert if there are problems.